src/Search.js

```diff
@@ -17,7 +18,7 @@ const Search = () => {
                 className="form-control"
                 placeholder="Customer Id"
               />
-              <button className="btn btn-primary">Search IDs</button>
+              <Button text={"Search IDs"}/>
```

Comment:
This isn't incorrect, but if you're just passing a string as a prop, then the curly braces are unnecessary: `<Button text="Search IDs: />` works fine too. And it's 2 less characters to type :)